notebooks/ImvizExample.ipynb

```diff
@@ -395,7 +406,7 @@
    "outputs": [],
    "source": [
     "# Center the image on given pixel position.\n",
-    "viewer.center_on((1173, 1013))  # X, Y (0-indexed)"
+    "viewer.center_on((1900, 2500))  # X, Y (0-indexed)"
```

Comment:
Just running through the notebook, I'm getting an error on Imviz saying these coords are "Out of Bounds". Anyone else?

Reply:
Ahh good call, this will fail if you've blinked and have data `B` active because it's smaller. I'll add a note about that.